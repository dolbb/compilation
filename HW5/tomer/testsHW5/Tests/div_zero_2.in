void main()
{
    int bb = 5 / 0;
}
