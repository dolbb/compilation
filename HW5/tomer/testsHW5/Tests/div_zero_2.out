Loaded: ./exceptions.s
Error division by zero
